zmq.nim

```diff
@@ -57,7 +57,7 @@
 {.deadCodeElim: on.}
 when defined(windows):
   const
-    zmqdll* = "zmq.dll"
+    zmqdll* = "(lib)zmq.dll"
```

Comment:
Shouldn't this be `` "(lib|)zmq.dll"`` ?

Reply:
I had another look at https://nim-lang.org/docs/manual.html#foreign-function-interface-dynlib-pragma-for-import and yes you're right, sorry.  
I'll change it.